File: Python/Directory/remove_directory.py
import shutil

# Remove a directory.


def remove_directory(system_path):
    shutil.rmtree(system_path)


def main():
    # Remove a directory
    remove_directory("assets/remove/89mmtNQY7hM7389f48Sw46ZhbRDNQ2h9")


main()

File: Rust/src/Directory/remove_directory.rs
fn main() {
    // Remove a directory from the system.
    remove_directory("assets/remove/89mmtNQY7hM7389f48Sw46ZhbRDNQ2h9/");
}

// Remove a directory from the system.
fn remove_directory(directory_path: &str) {
    std::fs::remove_dir(directory_path).unwrap();
}
